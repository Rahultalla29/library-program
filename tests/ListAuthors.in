LIST AUTHORS
ADD COLLECTION sample1.csv
LIST AUTHORS
ADD COLLECTION sample1.csv
LIST AUTHORS
EXIT
