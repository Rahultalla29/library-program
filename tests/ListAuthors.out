user: No books in system.

user: 17 books successfully added.

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: No books have been added to the system.

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: Ending Library process.
